src/rgw/motr/gc/gc.cc

```diff
@@ -14,16 +14,59 @@
  */
 
 #include "motr/gc/gc.h"
+#include <cstdlib>
+#include <ctime>
 
 void *MotrGC::GCWorker::entry() {
   std::unique_lock<std::mutex> lk(lock);
   ldpp_dout(dpp, 10) << __func__ << ": " << gc_thread_prefix
     << worker_id << " started." << dendl;
 
+  // Get random number to lock the GC index.
+  uint32_t my_index = ceph::util::generate_random_number() % (motr_gc->max_indices);
+  // This is going to be endless loop
   do {
-
     ldpp_dout(dpp, 10) << __func__ << ": " << gc_thread_prefix
-      << worker_id << " iteration" << dendl;
+      << worker_id << " Iteration Started" << dendl;
+
+    std::string iname = "";
+    // Get lock on an GC index
+    int rc = motr_gc->get_locked_gc_index(&my_index);
+
+    // Lock has been aquired, start the timer
+    std::time_t start_time = std::time(nullptr);
```

Comment:
we may use `ceph::real_time` too, I was thinking to change it in motr_gc_obj_info as well.